### ISSUE TITLE
Resulting RoboImage has black background, although ActivityScenario background is set to TRANSPARENT

### ISSUE THREAD

**Issue by sergio-sastre** [takahirom/roborazzi#34]
Since Robolectric cannot render shadows, it is pretty useful to provide a background color to the activity launched via ActivityScenario to better distinguish the "borders", e.g. in CardViews.

AndroidUiTestingUtils offers that possibility like this (same for composables), for instance:
```kotlin
@get:Rule
val activityScenarioForViewRule =
        ActivityScenarioForViewRule(
            config = ViewConfigItem(...),
            backgroundColor = Color.TRANSPARENT,
        )
```

When used with Roborazzi, if backgroundColor = Color.TRANSPARENT, the background color is eventually black (I believe, it is might be set to null?). It works with other colours though

Moreover, I believe it is related to Roborazzi and not Robolectric, since if I use the following method `compareSnapshot`, instead of `captureRoboImage()`, the background is actually transparent...

```kotlin
fun compareSnapshot(view: View, name: String) {
    val image = view.drawToBitmap()
    val path = System.getProperty("user.dir")
    val file = File("$path/src/test", "$name.png")

    try {
        val bos = ByteArrayOutputStream()
        val out = FileOutputStream(file)
        out.write(bos.toByteArray());
        image.compress(Bitmap.CompressFormat.PNG, 100, out)
        out.flush()
        out.close()
    } catch (e: Exception) {
        e.printStackTrace()
    }
}
```

### Current result with TRANSPARENT backgroundColor:
With `captureRoboImage()`
![MemoriseViewHolder_Happy_full](https://user-images.githubusercontent.com/6097181/233208128-dd3a896b-63c6-427e-b43f-a12cf7f3a737.png)

With `compareSnapshot()`:
![MemoriseViewHolder_Happy_compare](https://user-images.githubusercontent.com/6097181/233207929-3426c111-e2a4-47a1-95f6-cb65e11316f6.png)

**Comment by takahirom** [comment]
Thank you for reporting this issue.
It could be due to this property that the library was designed for dumping layouts. As a result, this property offers good performance. However, I believe it is not suitable for taking screenshots.

https://github.com/takahirom/roborazzi/blob/409dc0026e416c8f9418114c47c5f3720d498ef0/roborazzi-painter/src/main/java/com/github/takahirom/roborazzi/RoboCanvas.kt#L21

**Comment by sergio-sastre** [comment]
Have not tested it, but according to the documentation, it seems you're correct. It does not render alpha, what is used for transparency:
[TYPE_USHORT_555_RGB](https://docs.oracle.com/javase/7/docs/api/java/awt/image/BufferedImage.html#TYPE_USHORT_555_RGB)
Represents an image with 5-5-5 RGB color components (5-bits red, 5-bits green, 5-bits blue) with no alpha.

It looks like the correct value would be [TYPE_4BYTE_ABGR](https://docs.oracle.com/javase/7/docs/api/java/awt/image/BufferedImage.html#TYPE_4BYTE_ABGR)
I would expect this as default value, but could be nice to allow to pass your own in the `captureRoboImage()`

Moreover, this might be a breaking change I believe? Although breaking changes are expected in alpha-releases...

An alternative would be to provide an extra method
`captureRoboImageWithAlpha()` `captureAlphaRoboImage()` or the like...

**Comment by takahirom** [comment]
If having transparency (alpha) is completely preferred, then breaking changes are inevitable, but it's a difficult decision since there is a trade-off in performance.

**Comment by takahirom** [comment]
Perhaps a normal implementation would be as follows. It is difficult to decide what to set the default to, though.
`RoborazziOptions(captureType = CaptureType.Screenshot(RoborazziImageType.TYPE_USHORT_555_RGB))`

**Comment by sergio-sastre** [comment]
I've given this a second thought. I believe this is something required in some rare scenarios like the one I presented.

This seems to happen only when the background needs to be alpha. Otherwise, the alpha bits are painted with the background color of the underlaying Activity, usually black or white depndeing on the App theme.

Therefore, I'd say it's better to keep it as it is, but offer the option to change it for scenarios as the one I described.

All that said, I'd prefer to have that option also in the captureRoboImage() method, not to make it dependent on the RoborazziTestRule, which requires a composeRule. Such a compose rule is not always necessary, for instance, in AndroidView tests
